tests/commands/test_bump_recipe.py

```diff
@@ -0,0 +1,59 @@
+"""
+:Description: Tests the `bump-recipe` CLI
+"""
+
+from click.testing import CliRunner
+from pyfakefs.fake_filesystem import FakeFilesystem
+
+from conda_recipe_manager.commands import bump_recipe
+from conda_recipe_manager.commands.utils.types import ExitCode
+from tests.file_loading import get_test_path
+from tests.smoke_testing import assert_cli_usage
+
+
+def test_usage() -> None:
+    """
+    Smoke test that ensures rendering of the help menu
+    """
+    assert_cli_usage(bump_recipe.bump_recipe)
+
+
+def test_bump_recipe_cli(fs: FakeFilesystem) -> None:
+    """
+    Test for the case when build number is successfully incremented by 1.
+    :param fs: pyfakefs fixture used to replace the file system
```

Comment:
```suggestion

    :param fs: `pyfakefs` Fixture used to replace the file system
```
Otherwise the docs won't render correctly.